setupext.py

```diff
@@ -61,6 +61,16 @@ def _get_xdg_cache_dir():
     'basedirlist': None
     }
 
+compile_args = []
+link_args = []
+# macOS >= 10.14 does not provide libstdc++, so we have to manually specify
+# libc++ in a number of places
+if sys.platform == 'darwin':
```

Comment:
I wonder if this should actually be changed to a check for Xcode 10 instead of the OS version? (if that's even possible?)

Reply:
Thanks for the tip! I think this should be working now.